keps/sig-apps/2232-jobs-stopped/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: KEP Template
+kep-number: 2232
+authors:
+  - "@adtac"
+owning-sig: sig-apps
+status: provisional # provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
```

Comment:
Hi there, 1.21 Enhancements Lead here.
One of the requirements for the KEP is to have an `implementable` status before the enhancements freeze. Please keep that in mind and make sure if `provisional` needs to be changed or not for this PR. thank you!

Reply:
Thanks Anna, status is now updated to implementable.